### PR TITLE
Revamped instructions for week one, with no Ruby/server-side stuff

### PR DESCRIPTION
**Pull request by RobertAKARobin**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -42,22 +54,3 @@ Personal websites from some previous WDI grads:
 
 You can also Google "developer portfolio website" for more ideas :)
 
-### Domains
```
Comment (jshawl) [inline readme.md:45]: should keep the below, label it as bonus

File: readme.md [hunk 2]
```diff
@@ -1,22 +1,48 @@
...
+- Your HTML should [pass the W3 HTML validator](https://validator.w3.org/#validate_by_input).
+- Your CSS should [pass the W3 CSS validator](https://jigsaw.w3.org/css-validator/#validate_by_input).
+- Javascript isn't required, but you get bonus points for giving it a shot!
+
+## Pairing with Others
+
+Every student must create their *own* website, but you're **welcome and encouraged** to collaborate with others! The best way to create a great website is to get targeted feedback from others at several points in the process.
 
-### Building it
+Part-way through the day, we'll ask you to pair with another student and review each others' sites:
```
Comment (jshawl) [inline readme.md:21]: please add specific time

**Comment by RobertAKARobin** [comment]
@jshawl Responded!

**Comment by jshawl** [comment]
:+1:

**Comment by amaseda** [comment]
:+1:

**Pull request merged**